cpp/ql/src/Likely Bugs/Leap Year/UncheckedReturnValueForTimeFunctions.ql

```diff
@@ -1,5 +1,5 @@
 /**
- * @name Year field changed using an arithmetic operation is used on an unchecked time conversion function
+ * @name Unchecked return value for time conversion function
  * @description A year field changed using an arithmetic operation is used on a time conversion function, but the return value of the function is not checked for success or failure
```

Comment:
The `@description`s of the two queries still match, and to be honest, I find the names and descriptions of these queries quite confusing (compared to the violation messages and qhelp, which make things much clearer).

Reply:
I've pushed a commit to try and improve these.